flag_engine/utils/json/encoders.py

```diff
@@ -8,7 +8,7 @@ class DecimalEncoder(json.JSONEncoder):
     int/float(for us) converted to decimal by boto3/dynamodb.
     """
 
-    def default(self, obj):
+    def default(self, obj: object) -> object:
```

Comment:
I think this one should return a generic JSON value type

Reply:
I felt having to define the usual `JsonType = Union[None, int, str, bool, List[JsonType], Dict[str, JsonType]]` here was way too verbose for our goals here.

I'm happy to add this if your opinion is stronger.

Reply:
Well, it's not a big deal, but on projects I've worked on there was a `JsonType` defined in on place and used all over the application. It seems verbose, but it's not really *that* big. Most objects are not going to be included in the type. But like I said, no showstoppers. Totally your call.

Reply:
I think I understand where you're coming from here. However, this is the only place for this type annotation for now — and, I imagine, for a long time as we strive to remove any serialisation logic from the engine. I believe it's too much cognitive load just for this one occurrence.

The `object` return type is truthful and gets the job done nicely here, IMO. I'll be happy to revise this annotation should we add some other JSON manipulating code to the library in the future.